docs/setup/settings/servers.md

```diff
@@ -168,34 +169,34 @@ This only needs to be run when the system indicates "There is a pending update o
+This is accomplished by the User Agent Client (UAC), or a soft phone / hard phone, connecting to the ConnexCS server with a `REGISTER` message which includes their current location and their username including some extra information.
 
 The information is then stored, and made available as a lookup. When someone calls a user, it routes the call to their IP address.
 
-### NAT Keep Alives
+### Network Address Translation (NAT) Keep Alives
 
 When a user endpoint registers with UDP via NAT, a port mapping is created. However, the NAT will time out (and the call will disconnect) if the connection isn't maintained keep-alive by traffic.
 
 ConnexCS will periodically send keep-alive messages to the registered UAC, as either UDP or full SIP `OPTIONS` messages.
 
 While both work in this scenario, the SIP ```OPTIONS``` ping will reply to the ConnexCS Server. From this ping, we can monitor the latency of the connection and discern if NAT has destroyed the connection or if the user has gone offline without notifying the system.
 
-Once a user is registered via UDP, the connection is verified once again.
+Once a user registers via UDP, the connection is verified once again.
 
 ### Deployment Options
 
 **1. Single Server**: No additional configuration is needed for a single server.
-**2. Server Array**: For multiple servers, decide whether users registered to *Server A* can call users on *Server B*. If so, set the **UAC Location Array Sharing** option (found under [**Server Config**](https://docs.connexcs.com/setup/settings/servers/#server-config)) for the servers which will make the registrations.
-**3. Server Cluster**: Servers in a cluster share all user location registrations. This is the best solution for scaling to many thousands of registrations.
+**2. Server Array**: For various servers, decide whether users registered to *Server A* can call users on *Server B*. If so, set the **UAC Location Array Sharing** option (found under [**Server Config**](https://docs.connexcs.com/setup/settings/servers/#server-config)) for the servers which will make the registrations.
+**3. Server Cluster**: Servers in a cluster share all user location registrations. This is the best solution for scaling to thousands of registrations.
 
 !!! note "You won't be able to route calls to users registered against any servers outside the cluster."
 
 !!! tip "Ping Overhead"
-    Due to design constraints that occur with multiple servers in an array, NAT Keep Alive Pings can be quite intensive. You could use *Disable UAC Ping* from the server page, upgrade to a cluster, or use a single registration server to solve this.
+    Due to design constraints that occur with various servers in an array, NAT Keep Alive Pings can be quite intensive. You could use *Disable UAC Ping* from the server page, upgrade to a cluster, or use a single registration server to solve this.
```

Comment:
Various is not a suitable replacement for multiple in this sentence.

Reply:
Replaced **various** with **multiple**